src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -351,6 +341,7 @@ public static function get(int $id): array
 
     /**
      * Get the comments
+     * @deprecated
```

Comment:
if it is deprecated the entire functionality should be in the replacement

Reply:
It is never used, and I don't see any at this moment.

Reply:
ok